parm/metplus_config/nfcens/wave_grid2obs/plots/py_plotting_wave.config

```diff
@@ -108,7 +108,8 @@ fi
 export LOG_LEVEL="DEBUG"
 
 # Version of MET listed in the metplus .stat files used to create graphics
-export MET_VERSION="11.0"
+#export MET_VERSION="11.0"
```

Comment:
@SamiraArdani-NOAA Can you please remove the commented out line? Thanks!

#export MET_VERSION="11.0"